microshift_configuring/microshift-kubeconfig.adoc

```diff
@@ -0,0 +1,17 @@
+:_content-type: ASSEMBLY
+[id="microshift-cluster-access-kubeconfig"]
+= Cluster access with kubeconfig
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-networking
+
+toc::[]
+
+Learn about how `kubeconfig` files are used with {product-title} deployments. CLI tools use `kubeconfig` files to communicate with the API server of a cluster. These files provide cluster details, IP addresses, and other information needed for authentication.
+
+include::modules/microshift-kubeconfig-overview.adoc[leveloffset=+1]
+include::modules/microshift-kubeconfig-local-access.adoc[leveloffset=+1]
+include::modules/microshift-accessing-cluster-locally.adoc[leveloffset=+2]
+include::modules/microshift-kubeconfig-remote-con.adoc[leveloffset=+1]
+include::modules/microshift-kubeconfig-generating-remote-kcfiles.adoc[leveloffset=+1]
+include::modules/microshift-accessing-cluster-open-firewall.adoc[leveloffset=+2]
+include::modules/microshift-accessing-cluster-remotely.adoc[leveloffset=+2]
```

Comment:
The [modular docs assembly template](https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_ASSEMBLY_a-collection-of-modules.adoc) says that there should be a blank line after each `include::` statement. I know that not all of our assemblies follow that guidance, but you might want to fix it before merge.